.vib/matomo/goss/goss.yaml

```diff
@@ -0,0 +1,26 @@
+file:
+  /opt/bitnami/matomo/js:
+    filetype: symlink
+    linked-to: /bitnami/matomo/js
+    exists: true
+  /opt/bitnami/matomo/plugins:
+    filetype: symlink
+    linked-to: /bitnami/matomo/plugins
+    exists: true
+  /bitnami/matomo/config/config.ini.php:
+    mode: "0644"
+    filetype: file
+    exists: true
+# Checks the user running the container is non-root
+command:
+  user-id-test:
+    exec: if [ "$(id -u)" -eq 0 ]; then exit 1; fi
+    exit-status: 0
+    stdout: []
+    stderr: []
+  which-php:
+    exec: which php
```

Comment:
We can leave this, or we can check if  Apache httpd process is running using `ps aux`. 
I would say it's up to you.